lib/srv/reexec.go

```diff
@@ -1430,3 +1467,37 @@ func (o *osWrapper) newParker(ctx context.Context, credential syscall.Credential
 
 	return nil
 }
+
+func initLogger(name string, cfg ExecLogConfig) {
+	logWriter := os.NewFile(LogFile, fdName(LogFile))
+	if logWriter == nil {
+		return
+	}
+
+	fields, err := logutils.ValidateFields(cfg.ExtraFields)
+	if err != nil {
+		return
+	}
+
+	var logger *slog.Logger
```

Comment:
Do we need this logger? It doesn't look referenced outside of the two cases where we could define a locally scoped logger instead.
```suggestion
```